setup.py

```diff
@@ -59,14 +59,20 @@
         'nbclean==0.3.2',
         'nbsphinx==0.8.8',
     ],
+    'proxy': [
+        'jupyter-server-proxy>=1.5.2'
+    ]
 }
 
 extras_require['dev'] = [
+    'jupyterlab==3.1.14',
     'jupyter_server==1.11.0',
     'traitlets==4.3.3',
     'markupsafe==2.0.1',
 ] + extras_require['testing'] + extras_require['docs']
 
+extras_require['all'] = extras_require['dev'] + extras_require['proxy']
```

Comment:
Perhaps an unintuitive name, but the convention I've seen with the `all` extras_require is to capture all optional runtime dependencies for the package (not development dependencies). e.g., an easy way to install `vitessce` and anything that might raise an `ImportError` by an end user in normal use.

```bash
pip install vitessce[all] # installs vitessce, proxy, widget, etc
```

And then `dev` would include additionally include `all` to ensure that any developer has all runtime dependencies and development dependencies.

Reply:
Ah i see, that makes sense, I did not realize the convention